go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/onlineconf/onlineconf-go
 go 1.8
 
 require (
+	github.com/alldroll/cdb v1.0.3
```

Comment:
`onlineconf-updater` uses https://github.com/colinmarc/cdb to write CDB files.
It will be better to use the same module here if possible.